packages/3id-did-resolver/src/index.ts

```diff
@@ -194,6 +194,7 @@ const resolve = async (
   }
   const resp = await ceramic.multiQuery(query)
 
+  if (!resp[didId]) throw new Error(`Failed to properly resolve 3ID, stream ${didId} not found in response.`)
```

Comment:
@oed we should probably do the same thing for accessing the commitId below on line 200, right?

Reply:
oh nevermind, I see it's already prepared for that to be undefined by using `?`